scripts/solve.js

```diff
@@ -217,7 +218,8 @@ async function runStaticAnalysis(code) {
     difficulty: `The Halstead difficulty is ${analysis.difficulty}.
       ${generateTooltipHTMLString(DIFFICULTY_DESC, 'right')}`,
     lloc: `${analysis.lloc} logical lines of code.
-      ${generateTooltipHTMLString(LLOC_DESC, 'right')}`
+      ${generateTooltipHTMLString(LLOC_DESC, 'right')}`,
+    bigOnotation: ``
```

Comment:
what is the purpose of the big o notation description?

Reply:
This was not supposed to be on this PR